### DISCUSSION TITLE
WinML or DirectML support planned?

### DISCUSSION THREAD

**Discussion by oscarbg** [apache/incubator-mxnet#10456] [Q&A]
Windows only, but this should support AMD and Intel GPUs using D3D12 backend or even DirectML which with custom metacommands seems even able to use tensor cores on Volta GPUs..
Hope to hear news at least by Build conference in May

**Comment by Roshrini** [comment]
@sandeep-krishnamurthy Can you add label Question, FeatureRequest

**Comment by oscarbg** [comment]
just a note I'm asking about Win32 classic API part of WinML (not the UWP Net & C#)  WinML is a Win32 API as shown in Windows insider SDKs and this month 1803 SDK it has C++ header WinML.h and WinML.lib.. altough still no samples using Win32 API presumably at Build some Win32 API samples released..
See how Win32 API:
![winml](https://user-images.githubusercontent.com/669305/38595578-f43b381a-3d4c-11e8-84d4-5f55b0b6241b.jpeg)

Taken from (with DirectML details):
https://blogs.msdn.microsoft.com/directx/2018/03/19/gaming-with-windows-ml/
